assets/src/scripts/web-services/models.spec.js

```diff
@@ -107,16 +109,15 @@ describe('Models module', () => {
 
     describe('getPreviousYearTimeSeries', () => {
         const siteID = '05413500';
-
-        const startDate = new Date('2018-01-02T15:00:00.000-06:00');
-        const endDate = new Date('2018-01-02T16:45:00.000-06:00');
+        const startDate = 1514872800000; // milliSecond version of 01/02/2018
```

Comment:
Can we add a test specifically for leap year? This test would have passed before the change.

Reply:
I'll do my best